charts/presto/templates/deployment-worker.yaml

```
@@ -50,9 +52,9 @@ spec:
       {{- end }}
       containers:
         - name: worker
-          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
-          imagePullPolicy: {{ .Values.image.pullPolicy }}
-          {{- with .Values.worker.command }}
+          image: "{{ if .Values.velox.enabled }}{{ .Values.velox.image.repository }}:{{ .Values.velox.image.tag | default .Chart.AppVersion }}{{ else }}{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}{{ end }}"
+          imagePullPolicy: {{ if .Values.velox.enabled }}{{ .Values.velox.image.pullPolicy }}{{ else }}{{ .Values.image.pullPolicy }}{{ end }}
+          {{- with .Values.worker.command }}                        
```

The spaces seem to be redundant. If so, could you please delete them?
```suggestion
          {{- with .Values.worker.command }}
```